src/builder/create_embed.rs

```diff
@@ -101,6 +101,11 @@ impl CreateEmbed {
     }
 
     /// Set the image associated with the embed. This only supports HTTP(S).
+    ///
+    /// The URL must be ASCII alphanumeric characters ('a-z', 'A-Z', '0-9').
+    /// Underscores ('\_'), dashes ('-') and dots ('.') are allowed but they must not
+    /// appear as the first character. Otherwise, the attachment will be sent separately from the
+    /// Embed.
```

Comment:
This is a bit of a misfit with the line above saying it only supports HTTP(S), either adjust that line or adjust what you have written.